config/routes.rb

```diff
@@ -1,15 +1,17 @@
-Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
-
-  root "users#index"
-
-  resources :users, only: [:show, :index] do
-    resources :posts, only: [:show, :index] do
-      # Paginate post with kaminari
-      get '/page/:page', action: :index, on: :collection
-    end
-  end
-end
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+
+  root "users#index"
+
+  resources :users, only: [:show, :index] do
+    resources :posts, only: [:show, :index, :new, :create] do

```

Comment:
- [ ] For this Milestone, we are requested to use only two pair of routes(:new and :create) corresponding to the `#new` and `#create` posts controller actions
![Screenshot from 2023-10-06 21-57-28](https://github.com/henask12/Blogify/assets/63087491/c58ef0c7-fcdb-4356-b90e-6f1709406bc2)